### PR TITLE
add colors and sizes variables

### PR DESCRIPTION
**Pull request by fmmricardo**
What?
* adds colors variables (names based on coolors website)
* adds size variables (modular scale : base font- 20px; scale ratio - 1,333)

### CODE REVIEW

**Review by joaojusto — APPROVED** [review]

File: postcss.config.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
 module.exports = {
   plugins: [
-    "postcss-import",
-    "postcss-nested",
-    "postcss-flexbugs-fixes",
+    'postcss-import',
```
Comment (jfranciscosousa) [inline postcss.config.js]: Why the single quotes :see_no_evil:

Comment (fmmricardo) [reply postcss.config.js]: @jfranciscosousa , I think that's my prettier. As Pedro said, here we go 😅...

**Review by jfranciscosousa — APPROVED** [review]
Just fix prettier stuff and it's ok :+1:

File: postcss.config.js (hunk 1 above)
Comment (fmmricardo) [reply postcss.config.js:3]: @jfranciscosousa , I think that's my prettier. As Pedro said, here we go 😅...

**Pull request merged**